### DISCUSSION TITLE
Is there any implementation or road map available on incorporation chat suggestions?

### DISCUSSION THREAD

**Discussion by AlbertoPerez8** [Azure/communication-ui-library#3349] [Q&A]
I'm part of a team that has been developing a chat application that uses the statefull chat components provided from the communication library, and we had thoughts on adding a chat suggestion function similar to that of those in Teams chats. 

![image](https://github.com/Azure/communication-ui-library/assets/36648131/ed364729-5651-4319-9421-7ec99d54651e)

Is there any available functionality already incorporated into Azure Communication Services, or some kind of road map / suggested demos that can be used to add the functionality directly from our client? Are there any resources on ways to incorporate an artificial intelligence models like chatGPT to automatically generate appropriate suggestions based on the conversation flow in the case there's no easy way to implement from ACS?

**Comment by AmyL219** [comment]
Thank you for your inquiry about incorporating AI functionalities. We are actively exploring such integrations. However, we haven't yet established a definitive timeline for the introduction of this feature.

While our team is focusing on enhancing ACS with AI capabilities, we encourage you to explore how you might leverage [Azure AI Service](https://learn.microsoft.com/en-us/azure/communication-services/concepts/chat/concepts#build-intelligent-ai-powered-chat-experiences) to augment your ACS Chat experience. These [AI Services](https://learn.microsoft.com/en-us/azure/ai-services/) offer a variety of services that can potentially enhance your application and provide automatic suggestions based on conversation flow.
